tests/acceptance/stepDefinitions/filesContext.js

```diff
@@ -189,6 +189,19 @@ Then('file/folder {string} should be listed in the trashbin on the webUI', funct
   return client.page.FilesPageElement.filesList().waitForFileVisible(file)
 })
 
+Then('the deleted elements should be listed in the trashbin on the webUI', function () {
+  return assertDeletedElementsAreListed()
+})
+
+Then('file {string} should be listed in the trashbin folder {string} on the webUI', function (file, folder) {
```

Comment:
same here, in phoenix we can use the step that is also use for the main file list